### ISSUE TITLE
In Comboboxes, listbox lacks required ID for aria-controls mapping

### ISSUE THREAD

**Issue by Webhac** [ng-select/ng-select#2800]
### Describe the bug

**Description:**
The aria-controls attribute contains a value that does not reference a valid ID. The listbox should be assigned an ID, and that ID should be referenced in the aria-controls attribute on the <input> element of the combobox.

For specifications, refer to:
https://www.w3.org/TR/wai-aria-1.2/#combobox
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/combobox_role

**Actual Result:**
The ID referenced in aria-controls is not defined on the listbox.

### Steps to Reproduce

Go to https://ng-select.github.io/ng-select/#/data-sources
Inspect the combobox under “Backend data with async pipe.”
In the DOM, locate the aria-controls attribute on the <input> element.
Note the value referenced in aria-controls.
Inspect the listbox container and look for a matching ID.
Observe that the listbox does not have the ID referenced in aria-controls.

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

https://ng-select.github.io/ng-select/#/data-sources

### Expected behaviour
Move the aria-controls id from the dropdown panel to the listbox element. 
The listbox should be assigned an ID, and that ID should be referenced in the aria-controls attribute on the <input> element of the combobox. 

### What version of the library you see this issue?

https://ng-select.github.io/ng-select/#/data-sources

### What version of Angular are you using?

Angular 20

### Screenshots

<img width="1903" height="977" alt="Image" src="https://github.com/user-attachments/assets/898a43c8-7358-418e-875e-fc9aabe2b834" />

### Desktop? Please complete the following information

Windows 11
Chrome Version 147.0.7727.138

**Comment by kismetcalleio** [comment]
This is affecting me as well. ANDI and WAVE tests do not pass. 

<img width="1120" height="866" alt="Image" src="https://github.com/user-attachments/assets/ff5ca04f-9e1d-4bf3-bd99-d86278a58660" />

<img width="1120" height="458" alt="Image" src="https://github.com/user-attachments/assets/4edc34c5-2a8d-439c-82b5-883b11183ebd" />

**Comment by gtayloraamc** [comment]
Axe DevTools reporting the same issue.

<img width="1887" height="959" alt="Image" src="https://github.com/user-attachments/assets/cb62cc74-8cee-4aac-93cb-a0c73088ca90" />
